config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  devise_for :users
```

Comment:
Will this create all the routes?

Reply:
No, it won't generate all the routes, only the modules that we specify in User model
https://github.com/nimblehq/nimble-survey-web/pull/2/files#diff-4676c008b11a5480d73d4a6de01e45b9R4